xml/obs_build_config.xml

```diff
@@ -193,6 +193,9 @@ Keep: debbuild</screen>
            <para>Configure usage of ccache for build of packages mentioned by VALUE. Ccache package will be automatically
                installed and configured. The directory <filename>/.ccache/</filename> will be configured as cache dir.
                To configure ccache, the file <filename>/.ccache/ccache.conf</filename> can be modified as part of the build process by the <envar>$BUILD_USER</envar> environment variable. </para>
+
+           <para>if ccache is enabled but there is no archive present (eg: in case of newly branched packages, wiped binaries etc).
+           The archive of the, same-named package from repository paths configured in project's meta, will be used.</para>
```

Comment:
```suggestion
           The archive of the homonymous package from repository paths configured in the project's meta will be used.</para>
```

https://www.collinsdictionary.com/dictionary/english/homonymous

Reply:
sorry I force-pushed before looking here for suggestions. 

That is a good one thanks for that, however I am not sure how many others will have to look at dictionary for that word. :-)